client/jetpack-connect/install-instructions.jsx

```diff
@@ -0,0 +1,139 @@
+
+class InstallInstructions extends Component {
+	static propTypes = {
+		remoteSiteUrl: PropTypes.string.isRequired,
+	};
+
+	getInstructionsData() {
+		const { notJetpack, translate } = this.props;
+
+		return {
+			headerTitle: notJetpack
+				? translate( 'Ready for installation' )
+				: translate( 'Ready for activation' ),
+			headerSubtitle: translate( "We'll need you to complete a few manual steps." ),
+			steps: notJetpack
+				? [ 'installJetpack', 'activateJetpackAfterInstall', 'connectJetpackAfterInstall' ]
+				: [ 'activateJetpack', 'connectJetpack' ],
+			buttonOnClick: notJetpack ? this.installJetpack : this.activateJetpack,
+			buttonText: notJetpack ? translate( 'Install Jetpack' ) : translate( 'Activate Jetpack' ),
+		};
+	}
+
+	installJetpack = () => {
+		const { remoteSiteUrl } = this.props;
+
+		this.props.recordTracksEvent( 'calypso_jpc_instructions_click', {
+			jetpack_funnel: remoteSiteUrl,
+			type: 'install_jetpack',
+		} );
+
+		externalRedirect( addCalypsoEnvQueryArg( remoteSiteUrl + REMOTE_PATH_INSTALL ) );
+	};
+
+	activateJetpack = () => {
+		const { remoteSiteUrl } = this.props;
+
+		this.props.recordTracksEvent( 'calypso_jpc_instructions_click', {
+			jetpack_funnel: remoteSiteUrl,
+			type: 'activate_jetpack',
+		} );
+
+		externalRedirect( addCalypsoEnvQueryArg( remoteSiteUrl + REMOTE_PATH_ACTIVATE ) );
+	};
+
+	renderLocaleSuggestions() {
+		if ( this.props.isLoggedIn || ! this.props.locale ) {
+			return;
+		}
+
+		return <LocaleSuggestions path={ this.props.path } locale={ this.props.locale } />;
+	}
+
+	render() {
+		const { jetpackVersion, remoteSiteUrl } = this.props;
+		const instructionsData = this.getInstructionsData();
+
+		return (
+			<MainWrapper isWide>
+				{ this.renderLocaleSuggestions() }
+				<div className="jetpack-connect__install">
+					<FormattedHeader
+						headerText={ instructionsData.headerTitle }
+						subHeaderText={ instructionsData.headerSubtitle }
+					/>
+					<div className="jetpack-connect__install-steps">
+						{ instructionsData.steps.map( ( stepName, key ) => {
+							return (
+								<JetpackInstallStep
+									key={ 'instructions-step-' + key }
+									stepName={ stepName }
+									jetpackVersion={ jetpackVersion }
+									currentUrl={ remoteSiteUrl }
+									confirmJetpackInstallStatus={ this.props.confirmJetpackInstallStatus }
+									onClick={ instructionsData.buttonOnClick }
+								/>
+							);
+						} ) }
+					</div>
+					<Button onClick={ instructionsData.buttonOnClick } primary>
+						{ instructionsData.buttonText }
+					</Button>
+				</div>
+				<LoggedOutFormLinks>
+					<HelpButton />
+				</LoggedOutFormLinks>
+			</MainWrapper>
+		);
+	}
+}
+
+export default connect(
+	state => {
+		const remoteSite = getConnectingSite( state );
+		const remoteSiteData = remoteSite.data || {};
+		let notJetpack = ! remoteSiteData.hasJetpack;
+
+		const { installConfirmedByUser } = remoteSite;
```

Comment:
This logic is for the toggle link:

<img width="230" alt="screen shot 2018-03-13 at 11 24 04" src="https://user-images.githubusercontent.com/7767559/37339013-16dc736e-26b1-11e8-88c4-d5b1f47a3c65.png">